em_read_CAN.m

```diff
@@ -0,0 +1,69 @@
+% Copyright (c) 2023 Carleton University and National Research Council Canada

```

Comment:
Only `em_read_CAN` has the Carleton and NRCC copyright. Is this copyright required on all files created or modified by Carleton and NRCC?

Reply:
I will also add the copyright to Run_uncor_CA.m